modules/admin_manual/pages/enterprise/file_management/files_lifecycle.adoc

```diff
@@ -0,0 +1,126 @@
+= File Lifecycle Management
+:toc: right
+
+Manage the lifecycle of files within ownCloud to keep your working structure clean, organized and up to date. Choose an archiving and expiration policy that suits your workflow and select an appropriate archive destination to store your old files. Policies can be configured to include and exclude certain files, users
+and groups from the archiving actions. 
+
+== Installation & Configuration
+
+=== Upload Time Tracking
+
+The first part of file lifecycle management begins with knowing when files first enter ownCloud. To start tracking this the app must be enabled. This will store an `upload-time` with each file, which is the server time at which it first appeared on the ownCloud server. Only files are tracked, not folders since these are never archived, only the files within them.
+
+If installing this app on a system with existing data, the `lifecycle:set-upload-time` command can be used to pre-set a specific upload time to all existing files, so they may also be included in the archiving workflow. Files without an `upload-time` will not be considered for archive.
```

Comment:
installing - enabling ??

Reply:
Files without an upload time would be all files on a "normal" ownCloud instance without Lifecycle enabled or all files before lifecycle was enabled. This command is brought in to bulk set them.